File: app/src/main/java/com/supercasual/fourtop/ui/home/HomeViewModel.java
package com.supercasual.fourtop.ui.home;

import androidx.lifecycle.LiveData;
import androidx.lifecycle.MutableLiveData;
import androidx.lifecycle.ViewModel;

public class HomeViewModel extends ViewModel {

    private MutableLiveData<String> mText;

    public HomeViewModel() {
        mText = new MutableLiveData<>();
        mText.setValue("Раздел в разработке");
    }

    public LiveData<String> getText() {
        return mText;
    }
}

File: app/src/main/java/com/emikhalets/voteapp/view/home/HomeViewModel.kt
package com.emikhalets.voteapp.view.home

import androidx.lifecycle.ViewModel
import com.emikhalets.voteapp.data.AppRepository
import com.emikhalets.voteapp.data.AppRepository.Companion.get

class HomeViewModel : ViewModel() {
    private val repository: AppRepository?

    init {
        repository = get()
    }
}